### PR TITLE
Implemented pie chart to main page using google charts library

### PR DESCRIPTION
**Pull request by essencec**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,21 +7,20 @@
     <meta name="keywords" content="Essence Cain, portfolio, NCAT">
     <title>Essence Cain's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
+
```
Comment (tonymai) [inline portfolio/src/main/webapp/index.html:12]: you probably don't need this extra line break

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,6 +12,32 @@
...
 // limitations under the License.
 
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Hobby');
+  data.addColumn('number', 'Count');
+        data.addRows([
```
Comment (tonymai) [inline portfolio/src/main/webapp/script.js:22]: align indentation

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,6 +12,32 @@
...
+          ['Reading', 20],
+          ['Baking', 10],
+          ['Listening to Music', 15],
+          ['Watching Movies', 25],
+          ['Spending Time with Friends and Family', 30]
+        ]);
+
+  const options = {
+    'title': 'Hobbies',
+    'width':500,
```
Comment (tonymai) [inline portfolio/src/main/webapp/script.js:32]: space after the colon

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,6 +12,32 @@
...
+    'height':400
```
Comment (tonymai) [inline portfolio/src/main/webapp/script.js:33]: same

**Review by tonymai — APPROVED** [review]
Some minor formatting comments, but overall, it looks good!

**Review by ajw029 — APPROVED** [review]
LGTM

**Pull request merged**